src/extensions/mountainview/MountainView/MountainView.tsx

```diff
@@ -14,7 +14,9 @@ type AddViewAction = {
     type: 'AddView'
     pluginType: ViewPluginType
     plugin: ViewPlugin
+    label: string
```

Comment:
Before we were just using the plugin.label as the tab label. Now we explicitly provide it separately, so that we can use "Unit x" as the tab label when it is a MVSortingUnitView.